### PR TITLE
Bugfix/cx 5266 handle live_videos validation errors

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem

When a user takes a video that is very short (and by very short I mean you literally have to click through the video buttons really really fast) and submit it, the Onfido API returns a validation error:

```
{"attachment_file_size":["File size must be larger than zero"],"attachment":["File size must be larger than zero"]}
```

The SDK does not have the `attachment_file_size` error in the list of validation errors, therefore the user might not see any error (they just see the spinner) or they see the generic error message that implies that the connection was lost.

This error or any errors from the `video` element in the Confirm screen's previewer should be handled by the SDK and displayed to the user so that they can be guided to retake the video. 

# Solution

For all unhandled API errors or unexpected SDK UI errors, user should see the generic "please try again" message (i.e currently - connection lost)
- the generic "connection lost, please try again" message should be updated to just say "something's gone wrong, please try again"

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the TESTING_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [x] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Review by stefaniacardenas — APPROVED** [review]
LGTM!

File: src/components/Confirm/Confirm.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ const CALLBACK_TYPES = {
   video: 'onSubmitVideo',
   document: 'onSubmitDocument',
 }
+const UNEXPECTED_ERROR = 'REQUEST_ERROR'
```
Comment (ethan605) [inline src/components/Confirm/Confirm.js:33]: Minor: extra line break should be good here

```suggestion
const UNEXPECTED_ERROR = 'REQUEST_ERROR'

```

File: src/components/Confirm/Previews.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { h } from 'preact'
 import classNames from 'classnames'
-import { localised } from '../../locales'
+import { useLocales } from '../../locales'
```
Comment (ethan605) [inline src/components/Confirm/Previews.js:3]: I would say this is a good conversion, but seems a bit unrelated to the whole PR

File: src/components/CaptureViewer/CaptureVideoViewer.tsx [hunk 3]
```diff
@@ -1,24 +1,33 @@
...
   className,
   previewUrl,
+  onVideoError,
 }) => (
   <div className={classNames(style.videoWrapper, className)}>
     <video
       aria-label={ariaLabel}
       className={style.video}
       src={previewUrl}
+      onError={(event) => {
```
Comment (ethan605) [inline src/components/CaptureViewer/CaptureVideoViewer.tsx:25]: What's the default type of `event` here? Can we explicitly infer its type here to avoid (extra) casting inside the function?

**Review by ethan605 — CHANGES_REQUESTED** [review]
A minor suggestion and an open question

File: src/components/Confirm/Previews.js (hunk 2 above)
Comment (kopijunkie) [reply src/components/Confirm/Previews.js:3]: Ah, this is my bad. I was initially going to try to do a TS conversion but decided against it & left this behind. Will revert it to have only relevant changes in this PR.

File: src/components/CaptureViewer/CaptureVideoViewer.tsx (hunk 3 above)
Comment (kopijunkie) [reply src/components/CaptureViewer/CaptureVideoViewer.tsx:25]: In React I think it would have been `SyntheticEvent<HTMLVideoElement>` but Preact does not have Synthetic events so I didn't know of any other way to handle the `property not existing on EventTarget` TS error that was originally appearing without the extra casting.

Comment (ethan605) [reply src/components/CaptureViewer/CaptureVideoViewer.tsx:25]: Okay fair enough!

**Review by ethan605 — APPROVED** [review]
